itomate/itomate.py

```diff
@@ -181,10 +181,17 @@ async def activate(connection):
             continue
 
         # Set root path if it exists
-        if root_path:
-            for pane in tab_panes:
+        for pane in tab_panes:
+            if root_path:
```

Comment:
Prompt still looks to be under this conditional which is incorrect. This will make it so the prompt will only happen if root_path is set.. which we do not want